### PR TITLE
fix #116586: load sfz in background

### PR DESCRIPTION
**Pull request by hpfmn**
load sfz files in background and display a progress bar in the status bar

### CODE REVIEW

File: zerberus/zerberusgui.cpp [hunk 1]
```diff
@@ -173,7 +174,9 @@ void ZerberusGui::loadSfz() {
             QFuture<bool> future = QtConcurrent::run(zerberus(), &Zerberus::addSoundFont, sfName);
             _futureWatcher.setFuture(future);
             _progressTimer->start(1000);
-            _progressDialog->exec();
+            _progressDialog->open();
+            _progressBar = Ms::mscore->showProgressBar();
+            _progressBar->setFormat(tr("Loading sfz %p%"));
```
Comment (zhangchn) [inline zerberus/zerberusgui.cpp]: The last percent(%) in the format string should also be escaped!

**Comment by lasconic** [comment]
It's a good idea to use the MuseScore progress bar, so the UI is not blocked by a progress dialog. But from a code perspective, it's not really good to have Zerberus relying on MuseScore. It would maybe be better to send progress information in a signal and have someone listening to this information. For example, MuseScore could listen to it and display progress information in its progress bar. I close this PR, feel free to do a better one based on my comment  (or not :)

File: zerberus/CMakeLists.txt [hunk 2]
```diff
@@ -14,8 +14,23 @@ include (${PROJECT_SOURCE_DIR}/build/gch.cmake)
 
 QT5_WRAP_UI (zerberusUi zerberus_gui.ui)
 
+include_directories(
```
Comment (lasconic) [inline zerberus/CMakeLists.txt]: Not really sure why all of this should be added here.

File: zerberus/zerberusgui.h [hunk 3]
```diff
@@ -17,6 +17,7 @@
 #include "ui_zerberus_gui.h"
 #include "zerberus.h"
 #include <QDialogButtonBox>
+#include <QProgressBar>
```
Comment (lasconic) [inline zerberus/zerberusgui.h]: That's probably not needed.

**Pull request closed without merging**